SU-HSDO/suhumsci
# STN-523 : Vertical Linked Cards Spacing in Rows on Mobile

# READY FOR REVIEW

## Summary
The work in this PR fixes the spacing between rows of postcards on mobile screens. Spacing should be consistent between cards and rows of cards.

This PR also includes refactoring postcards to use grid `row-gap`. Previously only grid `column-gap` was used. Note that the shorthand property of grid `gap` implements both row and column gap.

Reference: [JIRA STN-523](https://sparkbox.atlassian.net/browse/STN-523)

## Need Review By (Date)
9/1/2020

## Urgency
medium

## Steps to Test
1. In your CLI, run `npm run test` and confirm our tests pass.
2. Review the vertical linked postcards at mobile screens on the following pages to confirm spacing has been fixed. Also review these pages at desktop screen width to confirm the spacing is consistent.
- [x] http://sparkbox-sandbox.suhumsci.loc/vertical-linked-cards-mobile
    * This page was created specifically for the testing of vertical linked cards at mobile
- [x] http://sparkbox-sandbox.suhumsci.loc/qa/qa-postcards
- [x] http://sparkbox-sandbox.suhumsci.loc/
- [x] http://sparkbox-sandbox.suhumsci.loc/qa/qa-page
- [x] http://sparkbox-sandbox.suhumsci.loc/qa/texts-cards-row-wells
- [x] http://sparkbox-sandbox.suhumsci.loc/qa/texts-cards-row-wells-hb-raised-cards
- [x] http://sparkbox-sandbox.suhumsci.loc/qa/texts-cards-row-wells-hb-stretch-vertical-linked-cards
- [x] http://sparkbox-sandbox.suhumsci.loc/qa/qa-raised-postcards
- [x] http://sparkbox-sandbox.suhumsci.loc/qa/qa-stretch-vertical-linked-postcards
